beacon-chain/rpc/prysm/v1alpha1/validator/proposer_test.go

```diff
@@ -2656,7 +2656,7 @@ func TestProposer_FilterAttestation(t *testing.T) {
 				HeadFetcher: &mock.ChainService{State: st, Root: genesisRoot[:]},
 			}
 			atts := tt.inputAtts()
-			received, err := proposerServer.validateAndDeleteAttsInPool(context.Background(), st, atts)
+			received := proposerServer.validateAndDeleteAttsInPool(context.Background(), st, atts)
 			if tt.wantedErr != "" {
```

Comment:
The error is removed from this signature, so should you delete the error handling things below?

Reply:
thanks, fixed